CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# How to Contribute
+
+We'd love to accept your patches and contributions to this project. There are
+just a few small guidelines you need to follow.
+
+## Contributor License Agreement
+
+Contributions to this project must be accompanied by a Contributor License
+Agreement. You (or your employer) retain the copyright to your contribution;
+this simply gives us permission to use and redistribute your contributions as
+part of the project.
```

Comment:
I would not give that away in here but refer to the exact legal terms in the CLA itself. Also, if the CLA would change we would need to think about updating this statement.

Reply:
I simply copied the Contributing guidelines from the current template. I will take your feedback and discuss this with @DavidPHirsch, if we want to tackle this now, or if we will generate a later task for improvements to this guideline file.

Reply:
also it seems like we have been using the contribution guidelines from googles repo: https://github.com/google/new-project/blob/master/docs/contributing.md

Reply:
we agreed to tackle this at a later stage, for now, we stick with the "google" default - and will improve this later with OSPO-37